skops/io/_dispatch.py

```diff
@@ -110,15 +110,17 @@ def _get_trusted(trusted, default):
 
         This is a convenience method called by child classes.
         """
+
         if trusted is True:
             # if trusted is True, we trust the node
             return True
 
         if trusted is False:
             # if trusted is False, we only trust the defaults
-            return default
+            _defaults = get_type_paths(default)
```

Comment:
I'm not sure where exactly to put this conversion. We should convert everything, including the `trusted` list here as well.

So maybe do `return get_type_paths(trusted)` as well?

Or should we do that inside `load` and `get_untrusted_set`?

Also, why not `return get_type_paths(defaults)`?